### ISSUE TITLE
Android build issue with `analytics-react-native-firebase`

### ISSUE THREAD

**Issue by kbrandwijk** [segmentio/analytics-react-native#155]
I'm using RN 0.60+. I get an error during build, both with manual linking and autolinking:

`> Project with path ':@segment/analytics-react-native' could not be found in project ':@segment_analytics-react-native-firebase'`

This is using version 1.1.1-beta.0.

**Comment by bsneed** [comment]
dupe of #154

**Comment by kbrandwijk** [comment]
It's not, of course, but I'm glad the PR fixes both issues...

**Comment by joaovpmamede** [comment]
@kbrandwijk did it fix for you? 
Still happening to me for the intercom integration with version `1.1.1-beta.2`

What versions are you using?

**Comment by ththugues44** [comment]
@joaovpmamede Here is a tip how to manage this, go to the android \ build.gradle file of the analytics-react-native-firebase module, change the line  ` api project(":@segment_analytics-react-native") ` with this  `api project(":segment_analytics-react-native")`

Then recompile your project after doing a clean gradlew

**Comment by kbrandwijk** [comment]
@joaovpmamede No I actually didn't try. I use these libraries professionally, and I'm not in a trial-and-error business, I can't afford that. So I'm sticking to my own fixed version.

I've mentioned before that I think it's ridiculous that a beta version is released as the official stable version to use for businesses, so I only update when I really, really, have to. This beta.2 only contains the fix I already made (but in a way that actually works), so I have no reason to update to it.

**Comment by bsneed** [comment]
@kbrandwijk that's rather unfortunate.  the spirit of open source is to help others.  If you're making fixes, and they're better than what others are doing, please do share them.

**Comment by kbrandwijk** [comment]
@bsneed Do not lecture me on the spirit of open source, you only need to look at the thousands of contributions in previous years on my profile to see that I understand the concept quite well, thank you very much.

Second, have a look at this page: https://segment.com/docs/connections/sources/catalog/libraries/mobile/react-native/. This library, that lives in the segment npm organisation, is the official, documented, way to use Segment with React-Native. It is _not_, as happens with many other projects, listed as: 'the community has come up with a couple of integrations you might find useful'. So in my opinion, for all intents and purposes, this is the _official_ package to use for a commercial product that my client (with a custom business account) pays more than good money for to use. 

Third, yes, I am making fixes. Let me take you through the process:
1. I run into an issue, that I trace down to something with an external library
2. I try and see if a bug has already been reported, or a PR is already underway.
3. If not, I always report the issue.
4. Not being able to wait for a fix in most cases, I patch the issue locally (I use `patch-package` for this).
5. Time permitting, I come back to the reported issue in the weekend, or when things are really busy, maybe the next weekend, to catch up on the issue status
6. If, at that point, a fix has not been made by the maintainers (who usually have a better view of the big picture), I will usually try and submit a PR. As you very well now, in most cases, that means more than just the fix. There might be test cases to be updated, some clean up work etc.. In other words, that takes time that I might or might not have after working 80 hours a week.

In this specific case, by the time I came back to the issue, a PR had already been proposed, and this issue was already being closed. At that point, no further contribution from my side is necessary.

So back to business. I already have a working situation on my project (I patched the issue locally, after all). So now I need to defend to my client, who is paying for my time, to spend time updating to a version that fixes the issue I had already fixed, introducing the risk of things breaking again if the fix was not done properly (such as in this particular case).

Unless there is a compelling reason to do so, the client is not going to approve this work. Usually, dependencies won't be updated again until there is a valid need to do say, because of new functionalities, other bug fixes, breaking API changes, etc. That's just how business works.

So while it is easy to call someone out on 'not contributing', the big picture is a little different.
